src/Tribe/Views/V2/iCalendar/Links/Link_Abstract.php

```diff
@@ -211,27 +234,50 @@ public function get_label( View $view = null ) {
 		 *
 		 * @return string $label The label that will be displayed.
 		 */
-		return apply_filters( "tec_views_v2_subscribe_links_{$slug}_label", $this->label, $this, $view );
+		return (string) apply_filters( "tec_views_v2_single_subscribe_links_{$slug}_label", $value, $this, $view );
 	}
 
 	/**
 	 * {@inheritDoc}
 	 */
-	public function get_single_label( View $view = null ) {
+	public function get_single_label( View $view = null ): string {
+		return $this->filter_get_single_label( $this->single_label(), $view );
+	}
+
+	/**
+	 * Returns the label for the single event view.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	abstract protected function single_label(): string;
```

Comment:
Same comment about adding abstract in already existing class.

Reply:
It adds a good value, if someone is extending they will know what they need to change it's ok.